activesupport/CHANGELOG.md

```diff
@@ -1,3 +1,21 @@
+*   ActiveSupport::Subscriber#attach_to now accepts an inherit_from: argument. When set to true,
+    it allows a subscriber to receive events for methods defined in the subscriber's ancestor class(es)
+
+    ```ruby
+    class StatsSubscriber < ActiveSupport::Subscriber
+      attach_to :active_record
+
+      def sql(event)
+        Statsd.timing("sql.#{event.payload[:name]}", event.duration)
+      end
+    end
+
+    class CustomStatsSubscriber < StatsSubscriber
+      attach_to :active_record, inherit_all: true
```

Comment:
I kind of feel like `inherit_all` should be the behavior because the way it works right now doesn't support methods that are included via modules or in this case inherited from the parent class.